docs/tools/cli/ticket.rst

```diff
@@ -0,0 +1,78 @@
+Command reference: ``avn ticket``
+==================================
+
+Here you'll find the full list of commands for ``avn ticker``.
+
+
+Create and manage support tickets
+------------------------------------
+
+Commands for managing Aiven support tickets.
```

Comment:
Should we add "To be able to manage support tickets your project should be covered by a support contract" 

I took "covered by a support contract" from the error message, though maybe there is a better definition of what project should be for the functionality to be supported.

Reply:
Added as a warning for now, I couldn't find any relevant material to link to. something for the future